core/jobs/batch_jobs/exp_validation_jobs.py

```diff
@@ -0,0 +1,201 @@
+# coding: utf-8
+#
+# Copyright 2022 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Validation Jobs for exploration models"""
+
+from __future__ import annotations
+
+from urllib.parse import urlparse  # pylint: disable-msg=C0003
```

Comment:
Why is the disable needed?

Reply:
without this the linter was giving a 'not a module error'

Reply:
Hmm, weird, maybe use the function from utils.py that uses `urlparse`? Or at least use disable that is easier to read.

Reply:
Done!